src/game/AI/ScriptDevAI/scripts/outland/shadowmoon_valley.cpp

```diff
@@ -615,8 +619,11 @@ struct npc_wildaAI : public npc_escortAI
             return;
 
         // all spirits follow
-        for (std::list<Creature*>::const_iterator itr = lSpiritsInRange.begin(); itr != lSpiritsInRange.end(); ++itr)
-            (*itr)->ForcedDespawn(6000);
+		for (std::list<Creature*>::const_iterator itr = lSpiritsInRange.begin(); itr != lSpiritsInRange.end(); ++itr)
+		{
+			(*itr)->ForcedDespawn(6000);
+			(*itr)->SetLevitate(true);
```

Comment:
I will have to look at this because SetLevitate does not work in TBC the way it works from wotlk+ But its only an issue in mangos-tbc core, not mangos-wotlk.